jupyter_core/command.py

```diff
@@ -123,7 +123,9 @@ def _execvp(cmd: str, argv: list[str]) -> None:
         if cmd_path is None:
             msg = f"{cmd!r} not found"
             raise OSError(msg, errno.ENOENT)
-        p = Popen([cmd_path] + argv[1:])  # noqa: S603
+        # Quoting path in Windows
+        cmd_line = f'"{cmd_path}"' + "".join(f' "{arg}"' for arg in argv[1:])
```

Comment:
This definitely will not be correct when `arg` contains quotes. Passing a list to Popen should result in proper quoting. If it doesn't, I think this is a bug in Python itself.

Instead of building quoted strings by hand, we should always use libraries like `shlex` to make sure it's done correctly or in a platform-independent way (in fact, that is why we pass a list to Popen, so I'm not sure what's going wrong here)

Can you check if `Popen(["some path", "with spaces"])` works for you without any Jupyer calls?